docs/en/changelog/migration_2_5.rst

```diff
@@ -0,0 +1,690 @@
+about behavioral changes and potential backwards compatibility breaks.
+Please review this document carefully when updating to Doctrine 2.5.
+
+First note, that with the ORM 2.5 release we are dropping support
+for PHP 5.3. We are enforcing this with Composer, servers without
+at least PHP 5.4 will not allow installing Doctrine 2.5.
+
+New Features and Improvements
+-----------------------------
+
+Events: PostLoad now triggered after associations are loaded
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Before Doctrine 2.5 if you had an entity with a ``@PostLoad`` event
+defined then Doctrine would trigger listeners after the fields were
+loaded, but before assocations are available.
+
+- `DDC-54 <http://doctrine-project.org/jira/browse/DDC-54>`_
+- `Commit <https://github.com/doctrine/doctrine2/commit/a906295c65f1516737458fbee2f6fa96254f27a5>`_
+
+Events: Add API to programatically add event listeners to Entity
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When developing third party libraries or decoupled applications
+it can be interesting to develop an entity listener without knowing
+the entities that require this listener.
+
+You can now attach entity listeners to entities using the
+``AttachEntityListenersListener`` class, which is listening to the
+``loadMetadata`` event that is fired once for every entity during
+metadata generation:
+
+.. code-block:: php
+
+    <?php
+    use Doctrine\ORM\Tools\AttachEntityListenersListener;
+    use Doctrine\ORM\Events;
+
+    $listener = new AttachEntityListenersListener();
+    $listener->addEntityListener(
+        'MyProject\Entity\User', 'MyProject\Listener\TimestampableListener',
+        Events::prePersist, 'onPrePersist'
+    );
+
+    $evm->addEventListener(Events::loadClassMetadata, $listener);
+
+    class TimestampableListener
+    {
+        public function onPrePersist($event)
+        {
+            $entity = $event->getEntity();
+            $entity->setCreated(new \DateTime('now'));
+        }
+    }
+
+Embeddedable Objects
+~~~~~~~~~~~~~~~~~~~~
+
+Doctrine now supports creating multiple PHP objects from one database table
+implementing a feature called "Embeddedable Objects". Next to an ``@Entity``
+class you can now define a class that is embeddable into a database table of an
+entity using the ``@Embeddable`` annotation. Embeddable objects can never be
+saved, updated or deleted on their own, only as part of an entity (called
+"root-entity" or "aggregate"). Consequently embeddables don't have a primary
+key, they are identified only by their values.
+
+Example of defining and using embeddables classes:
+
+.. code-block:: php
+
+    <?php
+
+    /** @Entity */
+    class Product
+    {
+        /** @Id @Column(type="integer") @GeneratedValue */
+        private $id;
+
+        /** @Embedded(class = "Money") */
+        private $price;
+    }
+
+    /** @Embeddable */
+    class Price
```

Comment:
There is a mismatch between the class name and the annotation on line 84. They should be the same, shouldn't they?